client/matching/client_gen.go

```diff
@@ -170,6 +170,25 @@ func (c *clientImpl) DispatchNexusTask(
 	return client.DispatchNexusTask(ctx, request, opts...)
 }
 
+func (c *clientImpl) ForceLoadTaskQueuePartition(
+	ctx context.Context,
+	request *matchingservice.ForceLoadTaskQueuePartitionRequest,
+	opts ...grpc.CallOption,
+) (*matchingservice.ForceLoadTaskQueuePartitionResponse, error) {
+
+	p, err := tqid.NormalPartitionFromRpcName(request.GetTaskQueuePartition().GetTaskQueue(), request.GetNamespaceId(), request.GetTaskQueuePartition().GetTaskQueueType())
```

Comment:
Actually this looks like it's generating the wrong code.. it's not referring to the PartitionId field at all so I don't see how it could work.
And DescribeTaskQueuePartition has the same problem.  @ShahabT @carlydf Did we test this with > 1 matching node?

Reply:
you are right, it should use PartitionFromPartitionProto instead.

I think there has been tests with multiple partitions, at least this one: TestAddMultipleTasksMultiplePartitions_ValidateBacklogInfo
But maybe no test with nonuniform partitions.

Reply:
Hahaha! So is DescribeTaskQueuePartition broken too? Its using the same generated code!

Reply:
yeah that was my bad. It seems I forgot that the `TaskQueuePartition.TaskQueue` field doesn't have the partition id in it, even after I literally created the `TaskQueuePartition` proto object in part so that the partition id would be more obvious..

Even as the author I fell victim to the point @dnr makes above about it being difficult to maintain two mental models about how to specify a task queue.

Initially I had called the field `TaskQueuePartition.TaskQueueName`, which would have made the lack of partition id in the name more obvious I think, but I changed the name to get some validation stuff to pass, which was a bit short-sighted.

I'll submit a PR to fix the issue in `DescribeTaskQueuePartition`

Reply:
This was fixed by 
https://github.com/temporalio/temporal/pull/6343

Reply:
Did you merge that one into your branch yet? The code still looks wrong here